### PR TITLE
Pair programming addition

### PR DESCRIPTION
**Pull request by davidobi**
Added 's' to ruby on rails

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -206,12 +210,97 @@ <h2 class="about-title">Skills</h2>
...
+                <div class="entry">
+                  <label for="name" class="input-label">Your name</label>
+                  <input
+                    class="input-text"
+                    type="text"
+                    name="name"
+                    id="name"
+                    maxlength="30"
+                    required
+                  />
```
Comment (mahbubul14) [inline index.html:246]: - [ ] I think you should add placeholder text in this `input` option. :writing_hand:

File: index.html [hunk 2]
```diff
@@ -206,12 +210,97 @@ <h2 class="about-title">Skills</h2>
...
+                </div>
+                <div class="entry">
+                  <label for="email" class="input-label">Your email</label>
+                  <input
+                    class="input-text"
+                    type="email"
+                    name="email"
+                    id="email"
+                    required
+                  />
```
Comment (mahbubul14) [inline index.html:256]: - [ ] I think you should add placeholder text for this input field. :writing_hand:

File: index.html [hunk 3]
```diff
@@ -206,12 +210,97 @@ <h2 class="about-title">Skills</h2>
...
+                  <label for="message" class="message-label"
+                    >Enter your message in here...</label
+                  >
+                  <textarea
+                    class="input-text"
+                    name="message"
+                    id="message"
+                    maxlength="500"
+                    required
+                  ></textarea>
```
Comment (mahbubul14) [inline index.html:268]: - [ ] I think you should add placeholder text in this `textarea` field. :writing_hand:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
# Required Changes ♻️

Hi @davidobi , 

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

### Mandatory Changes

- [ ] In your readme file, Please add the live demo link of your project. :link: 

- [ ] Pull Request should have a short summary describing introduced changes in more detail.

- [ ] Kindly check the comments under the review to see more. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Well implement the figma design.

### Optional Suggestions

- [ ] The commit message should be imperative & present tense.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (TesheMaximillan) [reply index.html:246]: According to Figma template-5, the placeholder is outside of the input and I put it in a label tag, and I gave it some style to make it as a static placeholder.

File: index.html (hunk 2 above)
Comment (TesheMaximillan) [reply index.html:256]: According to Figma template-5, the placeholder is outside of the input and I put it in a label tag, and I gave it some style to make it as a static placeholder.

File: index.html (hunk 3 above)
Comment (TesheMaximillan) [reply index.html:268]: According to Figma template-5, the placeholder is outside of the input and I put it in a label tag, and I gave it some style to make it as a static placeholder.

**Comment by TesheMaximillan** [comment]
# In this project

- A proper HTML form is used
- Styling is made according to the Figma template
- Pair Programming is made with my code partner
- Flex-Box is used to layout the elements

# After the review:

- I made a change in the README.md file

**Review by mateo951 — APPROVED** [review]
### Hi @davidobi,

Your project is complete!  There is nothing else to say other than... it's time to merge it ✔ 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**